tests/Application/config/packages/framework.yaml

```diff
@@ -4,3 +4,6 @@ framework:
     csrf_protection: true
     session:
         handler_id: ~
+    serializer:
+        mapping:
+            paths: ['%kernel.project_dir%/../../src/Resources/config/serialization']
```

Comment:
will it work like someone installs a plugin in a project? is there any description in readme?